.changeset/witty-seas-meow.md

```diff
@@ -0,0 +1,9 @@
+---
+'@aws-amplify/ui-vue': major
+---
```

Comment:
Since each framework has their own Authenticator texts that are corrected in this PR, I created a separate changeset for each one.